src/figuresClasses.ts

```diff
@@ -1,11 +1,78 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea: Function;
```

Comment:
[CODE STYLE] - The `getArea` function in the `Figure` interface should not accept any parameters. It should use object fields to calculate the area. Consider removing the `Function` type and replacing it with a more specific signature that implies the function will not accept parameters.